### PR TITLE
Init BottomNavigation

### PR DESCRIPTION
**Pull request by TranhGio**
## 
Screenshot_1568709827

### CODE REVIEW

File: app/src/main/res/menu/menu_main.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        android:icon="@drawable/ic_menu_opaque_24dp"
+        android:title="@string/genres"
+        app:showAsAction="ifRoom" />
+    <item
+        android:id="@+id/item_search"
+        android:icon="@drawable/ic_search_opaque_24dp"
+        android:title="@string/search"
+        app:showAsAction="ifRoom" />
+    <item
+        android:id="@+id/item_favourite"
```
Comment (thaohq-0814) [inline app/src/main/res/menu/menu_main.xml]:
```suggestion
        android:id="@+id/actionFavourite"
```

File: app/src/main/res/menu/menu_main.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        android:icon="@drawable/ic_home_opaque_24dp"
+        android:title="@string/home"
+        app:showAsAction="ifRoom" />
+    <item
+        android:id="@+id/item_genres"
+        android:icon="@drawable/ic_menu_opaque_24dp"
+        android:title="@string/genres"
+        app:showAsAction="ifRoom" />
+    <item
+        android:id="@+id/item_search"
```
Comment (thaohq-0814) [inline app/src/main/res/menu/menu_main.xml]:
```suggestion
        android:id="@+id/actionSearch"
```

File: app/src/main/res/menu/menu_main.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/item_home"
+        android:icon="@drawable/ic_home_opaque_24dp"
+        android:title="@string/home"
+        app:showAsAction="ifRoom" />
+    <item
+        android:id="@+id/item_genres"
```
Comment (thaohq-0814) [inline app/src/main/res/menu/menu_main.xml]:
```suggestion
        android:id="@+id/actionGenres"
```

File: app/src/main/res/menu/menu_main.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/item_home"
```
Comment (thaohq-0814) [inline app/src/main/res/menu/menu_main.xml]:
```suggestion
        android:id="@+id/actionHome"
```

File: app/src/main/res/layout/fragment_search.xml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/fragmentSearch"
```
Comment (thaohq-0814) [inline app/src/main/res/layout/fragment_search.xml]:
```suggestion
    android:id="@+id/layoutSearch"
```

File: app/src/main/res/layout/fragment_home.xml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/fragmentHome"
```
Comment (thaohq-0814) [inline app/src/main/res/layout/fragment_home.xml]:
```suggestion
    android:id="@+id/layoutHome"
```

File: app/src/main/res/layout/fragment_genres.xml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/fragmentGenres"
```
Comment (thaohq-0814) [inline app/src/main/res/layout/fragment_genres.xml]:
```suggestion
    android:id="@+id/layoutGenres"
```

File: app/src/main/res/layout/fragment_favourite.xml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/fragmentFavourite"
```
Comment (thaohq-0814) [inline app/src/main/res/layout/fragment_favourite.xml]:
```suggestion
    android:id="@+id/layoutFavourite"
```

**Comment by TranhGio** [comment]
Ready

**Pull request merged**